src/removeFromLocalStorage.test.js

```diff
@@ -0,0 +1,23 @@
+// const removeDataFromLocalStorage = require('./removeFromLocalStorage.js');
+
+test('remove item from local storage', () => {
+  const list = {
+    description: 'adda',
+    completed: false,
+    index: 1,
+  };
+
+  const list2 = {
+    description: 'kaaks',
+    completed: false,
+    index: 2,
+  };
+
+  const localstorage = JSON.parse(localStorage.getItem('list')) || [];
+  localstorage.push(list);
+  expect(localstorage).toHaveLength(1);
+  localstorage.push(list2);
+  expect(localstorage).toHaveLength(2);
+  localstorage.pop();
+  expect(localstorage).toHaveLength(1);
+});
```

Comment:
- [ ] Kindly make sure to test the function `removeDataFromLocalStorage()`. You commented the importation meaning that you are no longer testing the function. 
- [ ] You'll require to add tests for most pure functions
- [ ] Tests should be group using the description() method

Reply:
* function , removeDataFromLocalStorage() imported.
* Tests group using description method.
* Pure functions tested.